client/src/App.js

```diff
@@ -1,26 +1,40 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component} from 'react';
+import axios from 'axios';
+
+import NavBar from './NavBar';
+import Players from './Players/Players';
+
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+export default class App extends Component {
+constructor() {
+  super();
+  this.state ={
+    players: []
+  }
+}
+
+componentDidMount() {
+  axios
+  .get('http://localhost:5000/api/players/')
+  .then(response => {
+
```

Comment:
if you used a console log just comment it out and leave it for the future to show best practice